drivers/sensor/iis2dh/iis2dh_spi.c

```diff
@@ -100,28 +98,15 @@ stmdev_ctx_t iis2dh_spi_ctx = {
 int iis2dh_spi_init(const struct device *dev)
 {
 	struct iis2dh_data *data = dev->data;
+	const struct iis2dh_device_config *config = dev->config;
 
-	data->ctx = &iis2dh_spi_ctx;
-	data->ctx->handle = (void *)dev;
-
-#if DT_INST_SPI_DEV_HAS_CS_GPIOS(0)
-	/* handle SPI CS thru GPIO if it is the case */
-	data->cs_ctrl.gpio_dev = device_get_binding(
-		DT_INST_SPI_DEV_CS_GPIOS_LABEL(0));
-	if (!data->cs_ctrl.gpio_dev) {
-		LOG_ERR("Unable to get GPIO SPI CS device");
+	if (!device_is_ready(config->spi.bus)) {
```

Comment:
Change `device_is_ready` to `spi_is_ready`

Reply:
Thanks!